lib/pbio/drv/legodev/legodev_pup.c

```diff
@@ -405,6 +413,14 @@ PROCESS_THREAD(pbio_legodev_pup_process, ev, data) {
     PROCESS_END();
 }
 
+/**
+ * We get notified when the uart driver has completed sending or receiving data.
+ */
+static void uart_poll_callback(pbdrv_uart_dev_t *uart) {
+    // REVISIT: Only need to poll the specified uart device.
```

Comment:
We could add a flag that gets set here so that we only poke the uart instances that need it in the poll handler.